src/crimson/osd/pg_backend.h

```diff
@@ -63,6 +63,13 @@ class PGBackend
     uint32_t truncate_seq,
     uint32_t flags);
 
+  using checksum_errorator = ll_read_errorator::extend<
```

Comment:
OK, in that case the `osd_op_errorator` in `ops_executer.h` might need to be extended:

```cpp
  using osd_op_errorator = crimson::compound_errorator_t<
    call_errorator,
    read_errorator,
    write_ertr,
    get_attr_errorator,
    watch_errorator,
    PGBackend::stat_errorator,
    PGBackend::checksum_errorator>;
```

If it compiles without that, then all the errors `checksum_errorator` all already present in the `osd_op_errorator`.